cmd/agent.go

```diff
@@ -36,11 +36,31 @@ three agents into the distribution:
 	  throught the  gossip network
 	* Publisher agent: publish snapshots to the snapshot store`,
 	TraverseChildren: true,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I will declare the function outside, like in `server_start:RunE` just to be coherent